### PR TITLE
Update controller.java

### PR DESCRIPTION
**Pull request by atakanserifoglu**

### CODE REVIEW

File: controller.java [hunk 1]
```diff
@@ -2,13 +2,17 @@
 public class controller {
     public static void main(String[] args) {
         // Declare two numbers
-        int num1 = 10;
-        int num2 = 20;
+        int num1 = 40;
```
Comment (atakanserifoglu) [inline controller.java:5]: karmaşık

File: controller.java [hunk 2]
```diff
@@ -2,13 +2,17 @@
...
+        int num2 = 40;
 
         // Perform addition
         int sum = num1 + num2;
 
         // Print the result
         System.out.println("The sum of " + num1 + " and " + num2 + " is: " + sum);
     }
+
+    public static int add(int num1, int num2) {
```
Comment (tech-extremists) [inline controller.java:15]: wow

File: controller.java [hunk 3]
```diff
@@ -2,13 +2,17 @@
...
+        int num1 = 40;
+        int num2 = 40;
 
         // Perform addition
         int sum = num1 + num2;
 
         // Print the result
         System.out.println("The sum of " + num1 + " and " + num2 + " is: " + sum);
     }
+
```
Comment (tech-extremists) [inline controller.java:14]: aaa

File: controller.java [hunk 4]
```diff
@@ -2,13 +2,17 @@
 public class controller {
     public static void main(String[] args) {
         // Declare two numbers
-        int num1 = 10;
-        int num2 = 20;
+        int num1 = 40;
+        int num2 = 40;
```
Comment (atakanserifoglu) [inline controller.java:6]: numbers should be 30

Comment (atakanserifoglu) [inline controller.java:6]: numbers should be 30

Comment (atakanserifoglu) [inline controller.java:6]: numbers should be 30

Comment (atakanserifoglu) [inline controller.java:6]: !suggest numbers should be 30

Comment (atakanserifoglu) [inline controller.java:6]: !suggestnumbers should be 30

Comment (atakanserifoglu) [inline controller.java:6]: numbers should be 30 !suggest

Comment (Konuralpkilinc) [inline controller.java:6]: zencigot

Comment (Konuralpkilinc) [reply controller.java:6]: zwnfşro2

Comment (Konuralpkilinc) [inline controller.java:6]: change variable name to number2
```suggestion
        int number2 = 40;
```

**Pull request closed without merging**